interfaces/apparmor/spec.go

```diff
@@ -509,3 +517,12 @@ func (spec *Specification) GetUsesPtraceTrace() bool {
 func (spec *Specification) GetSuppressPtraceTrace() bool {
 	return spec.suppressPtraceTrace
 }
+
+// SuppressHomeIx to request explicit ptrace deny rules
+func (spec *Specification) SuppressHomeIx() {
+	spec.suppressHomeIx = true
+}
+
+func (spec *Specification) GetSuppressHomeIx() bool {
```

Comment:
As in the other PR I would suggest to discard `GetSupressesHomeIx` as this is only used by the back-end code with has access to the private attribute.

Reply:
I use the Getter in common_test.go and felt this was cleaner.